### PR TITLE
Fix offline upgrade download link

### PR DESCRIPTION
**Pull request by T1mL3arn**
In Firefox (92.0.1) `href="#"` does not allow to start downloading of `upgrade.html` (Chrome has no such problem). Making href completely empty fixes the issue.

### CODE REVIEW

File: plugins/tiddlywiki/upgrade/UpgradeWizard.tid [hunk 1]
```diff
@@ -50,7 +50,9 @@ For help and support, visit [[the TiddlyWiki discussion forum|http://groups.goog
 
 </div>
 
-//Your data will not leave your browser. <a href="#" download="upgrade.html">Download</a> this upgrader to use it offline//
+version <<version>>
+
```
Comment (T1mL3arn) [inline plugins/tiddlywiki/upgrade/UpgradeWizard.tid:54]:
```suggestion
```

**Comment by Jermolene** [comment]
Hi @T1mL3arn many thanks for your contribution. With apologies for the bureaucracy, please could you prepare a separate PR to the 'tiddlywiki-com' branch with your signature for the Contributor License Agreement (see [contributing.md](https://github.com/Jermolene/TiddlyWiki5/blob/master/contributing.md)).

**Comment by T1mL3arn** [comment]
> sign the CLA

Done.

**Comment by Jermolene** [comment]
Thank you @T1mL3arn

**Pull request merged**